### DISCUSSION TITLE
how to get Galaxy Brain in github

### DISCUSSION THREAD

**Discussion by adeemaamir240** [community/community#64016] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

how to get Galaxy Brain in GitHub

**Comment by lala-andy** [comment]
answer a question in community to get GitHub badge Galaxy Brain

**Reply by GhostOf0days** [reply]
This is partially wrong. @adeemaamir240 You have to answer 2 discussion posts where the original poster marks your reply as an answer. It may take a lot more than 2 total discussion posts answered if your replies are not marked as the answer. It depends on whether or not your replies are marked as the answer. 2 replies marked as an answer is the base amount you need. Bronze is 8 replies marked as an answer. Silver is 16 replies marked an an answer. Gold is 32 replies marked as an answer.

**Comment by Satyxm** [comment]
Writing at least 2 answers for the Discussion posts which the author marks as answers. 
@GhostOf0days said it right!

**Comment by stainlessdang** [comment]
I prefer you to read this article.
https://github.com/4xmen/Get-Github-Achievements/blob/main/assets/steps/README.galaxy-brain.md

**Reply by kuciapakrystian** [reply]
thank You for this link :)

**Comment by fauzirfann** [comment]
answer 2 question in Community Discussions

**Comment by dushmanta05** [comment]
I've recently received the Galaxy Brain achievement. The way to do this is by helping people with their questions in discussion. Let's say someone's answer from this particular discussion helped you; then you accept his or her answer as a solution. The person who posted that will be eligible to receive the Galaxy Brain achievement. Once that user gets two of his replies accepted as answers in the discussion, he'll get the Galaxy Brain achievement.So start engaging in discussion, and hopefully your reply will be marked as an answer.

**Comment by bryan-shea** [comment]
What's going on! Funny enough, I'm currently on the same quest to receive this achievement. I'll go ahead and include my outline below that I am currently following to try and get the Galaxy Brain achievement!

### **1.** Navigate to the Community: 
#### Visit the GitHub Community at https://github.com/community/community.

### **2.** Choose a Feedback Category: 
#### Select one of the available feedback categories, such as "Bug Reports," "Feature Requests," or "General Discussion."

- Example: If you're skilled in identifying software bugs, you might choose the "Bug Reports" category.

### **3.** Find Unanswered Questions: 
#### Browse through the selected category and look for unanswered questions that are awaiting responses.

- Example: You might find a question like "Why is this JavaScript function not working?" that hasn't received any answers. This is your chance to apply your expertise and provide assistance.

### **4.** Provide a Stellar Answer: 
#### Write a well-researched, accurate, and concise answer to the question. It's important to be clear and accurate as your comment will need to be selected as the submitted answer!

- Example: If the question is about a specific coding error, you could provide a detailed explanation of the mistake, offer a corrected code snippet, and include links to relevant documentation or resources.

### **5.** Repeat the Process: 
#### The Galaxy Brain achievement requires two submitted answers in any categories to unlock the Galaxy Brain achievement.

This achievement is a wholesome one as it gets new users, such as myself, acquainted with the community! Once you complete the above steps, its celebration time! You will find the Galaxy Brain achievement in your achievements list.

Let me know if this helps and good luck on obtaining the Galaxy Brain Achievement!

**Reply by dushmanta05** [reply]
Probably nothing but you've used the 1st heading of markdown and the comments look so big in GitHub mobile. 😅

**Reply by bryan-shea** [reply]
Ouch! haha my bad homie, lemme know if the updated reply looks any better? *jots down in his notepad to take into consideration the mobile viewers from now on*

**Reply by dushmanta05** [reply]
Hey Bryan! It looks great now. 😄

**Comment by ghostinhershell** [comment]
Thanks so much for posting in our Community @adeemaamir240. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
